src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -2004,6 +2010,8 @@ def aks_agentpool_update(cmd,   # pylint: disable=unused-argument
         instance.min_count = None
         instance.max_count = None
 
+    instance.tags = tags
```

Comment:
tags is None default, what if user doesn't pass `--tags` option, it will clear the tags by mistake right? But user doesn't want to clear tags right?
 I am not sure about the context, correct me if I misunderstand.

Reply:
No, it won't clear tags if no --tags passed. Only when customer input --tags "", the tags will be cleaned